example/runcard_example.py

```diff
@@ -0,0 +1,60 @@
+import os
+print("Sourcing runcard")
+dictCard = {
+            # folder       :   config file / runcard
+            "dummy_folder" : "config",
+}
+# for the example, keep everything in 'example' folder.
+exampleDir = os.path.dirname(os.path.realpath(__file__))
+
+## Run Variables
+
+# run-specific variables (overrides header.py variables)
+runcardDir = exampleDir
+# number of concurrent jobs to submit
+producRun = 5
+# ARC jobname to appear in squeue output
+jobName = "EXAMPLE_RUN"
+
+# ARC's jobs database: put this wherever you want
+arcbase = os.path.expanduser("~/.arc/jobs.dat")
+# pyHepGrid SQL database
+dbname = F"{exampleDir}/ARCdb_example.dat"
+
+
+# user-specific global variables (could go in user's header)
+copylog = True
+
+# automatically pick next unused seed
+import pyHepGrid.src.dbapi as dbapi
+baseSeed = dbapi.get_next_seed(dbname=dbname) + 1
+# baseSeed = 2173  # (setting seed manually)
+
+## Project-level Variables
+
+# Your custom subclass of `ProgramInterface` class in "src/pyHepGrid/src/program_interface.py"
+# import should be relative to this runcard
+runmode = "backend_example.ExampleProgram"
+
+runfile    = exampleDir+"/simplerun.py"
```

Comment:
The `runfile` path is put as is to the Dirac submission, so this wouldn't work since Dirac is then looking for the absolute path. E.g. I get the error:

```
Job Wrapper is starting execution phase for job 24221429
Failed to change mode to 775 for the executable /mt/home/mheil/grid_tools/pyHepGrid/example/simplerun.py
Failed to execute job Path to executable /mt/home/mheil/grid_tools/pyHepGrid/example/simplerun.py not found
Job failed in execution phase
Traceback (most recent call last):
  File "/data/condor_pool/dir_279935/DIRAC_ut5jqPpilot/job/Wrapper/Wrapper_24221429", line 145, in execute
    raise JobWrapperError((result['Message'], result['Errno']))
JobWrapperError: ('Path to executable /mt/home/mheil/grid_tools/pyHepGrid/example/simplerun.py not found', 0)
EXECUTION_RESULT[CPU] missing in sendJobAccounting 
```

Reply:
Should be fixed with b3b16fec3fd0f5ddd5dde97b13935feb4d9f6297